index.js

```diff
@@ -49,6 +58,15 @@ function formatDate(d) {
   return d.getFullYear() + pad2(d.getMonth() + 1) + pad2(d.getDate());
 };
 
+function sanitizeFileName(n) {
```

Comment:
Can we rename this variable to `filename`? (one-letter variable names are not generally used.)

Reply:
yep! I actually had changed it from `filename` to `f` lol. shouldn't have second guessed it